src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_ARRAY_CAPACITY = 16;
+    private static final int RESIZE_NUMBER = 2;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_ARRAY_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
+        if (size > (table.length * LOAD_FACTOR)) {
+            int capacity = DEFAULT_ARRAY_CAPACITY * RESIZE_NUMBER;
+            Node<K, V>[] newTable = (Node<K, V>[]) new Node[capacity];
 
+        }
+        size++;
     }
```

Comment:
The `put` method implementation is not complete. You've started to implement the resizing logic, but it's not finished. Also, the logic of putting the new node into the map is missing. Remember about the `Node` class parameterization.